src/pkg/services/m365/m365.go

```diff
@@ -55,34 +55,29 @@ func Users(ctx context.Context, acct account.Account, errs *fault.Bus) ([]*User,
 	return ret, nil
 }
 
-func UserIDs(ctx context.Context, acct account.Account, errs *fault.Bus) ([]string, error) {
+// UsersMap retrieves all users in the tenant, and returns two maps: one id-to-principalName,
+// and one principalName-to-id.
+func UsersMap(
```

Comment:
do SDK consumers need all the results this function has been modified to return? It may be easier to make a new function rather than force SDK consumers to update their calls